src/main/java/nl/windesheim/codeparser/analyzers/util/FilePartResolver.java

```diff
@@ -0,0 +1,84 @@
+package nl.windesheim.codeparser.analyzers.util;
+
+import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.Node;
+import nl.windesheim.codeparser.FilePart;
+
+import java.io.File;
+import java.util.ArrayList;
+
+/**
+ * Resolves FileParts.
+ */
+public final class FilePartResolver {
+
+    /**
+     *
+     */
+    private FilePartResolver() {
```

Comment:
Kan je de lege constructor verwijderen of er documentatie boven plaatsen?

Reply:
Nee, als je die weg haalt gaat codestyle over de zijk. Je moet een constructor hebben en die moet een javadoc hebben. :(